### DISCUSSION TITLE
how to do i update a interlinked schema

### DISCUSSION THREAD

**Discussion by rajamoulimallareddy** [prisma/prisma#16779] [Q&A] [answered]
```
model Guild {
  guildId       String        @id
  prefix        String        @default("t!")
  autoplay      Boolean       @default(false)
  djroles       String[]
  announcements Boolean       @default(false)
  setvc         String[]
  voiceId       String?
  textId        String? 
  requester     Boolean       @default(false)
  buttons       Boolean       @default(false)
  twenty4seven  Boolean       @default(false)
  language      String        @default("en")
  premium       PremiumModel? 
} 

model PremiumModel {
  id               Int     @id @default(autoincrement())
  active           Boolean @default(false)
  expiry           String?
  subtype_monthly  Boolean @default(false)
  subtype_lifetime Boolean @default(false)
  Guild            Guild?  @relation(fields: [guildGuildId], references: [guildId])
  guildGuildId     String? @unique
}```




how do I update the premiummodel using the guild schema?
can anyone show me an example?

**Comment by nurul3101** [comment, marked as answer]
Hello @rajamoulimallareddy 👋 

Here's an example of nested update based on your schema:
```typescript
import { PrismaClient } from '@prisma/client';

const prisma = new PrismaClient();

async function main() {
  // Create React
  await prisma.guild.update({
    where: {
      guildId: '123',
    },
    data: {
      announcements: true,
      autoplay: true,
      premium: {
        update: {
          active: true,
        },
      },
    },
  });
}

main()
  .catch((e) => {
    throw e;
  })
  .finally(async () => {
    await prisma.$disconnect();
  });
```
Here's a reference of [Nested Update](https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#nested-update-example). Let me know if you have any further questions.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
